api/services/passport.js

```diff
@@ -150,7 +155,13 @@ if (Features.enabled(Features.Flags.FEATURE_AUTH_UAA)) {
         ['pages.user', 'pages.support', 'pages.admin']
       );
 
-      if (!user) return callback(null, false, flashMessage);
```

Comment:
(Optional): I think there are two separate ways that this can fail: if they don't have a UAA profile sent back or if it isn't in the `pages.user` group. Is it worth logging these separately or will we see it in the `profile`?

Reply:
We can add additional logging in the UAA strategy and the verify UAA user. Should we set them as audit or error types for the events?

Reply:
I'd say audit since it we'll throw for failure later on

Reply:
Moved the event auditing into the `uaaStrategy` service to log if:
- UAA user is not in the verified group
- If no UAAIdentity exists in our DB that matches the UAA user
- If no User is associated to the UAAIdentity in our DB